docs/schema-replacement.md

```diff
@@ -0,0 +1,145 @@
+# Schema Replacement
```

Comment:
Can we also mention that no errors will be outputted as well? (Unless I missed this below!) :)

Reply:
Check Line  88...